src/sched/job_runner.h

```diff
@@ -57,6 +65,20 @@ class JobRunner : public std::enable_shared_from_this<JobRunner> {
 
     bool AddJob(std::function<void(JobAliveTokenPtr&&)>&& job, JobRunnerStrandPtr strand);
 
+    ///
+    /// If possible, run the job in the current thread immediately.
```

Comment:
What's the point of 3 slashes?

Reply:
This is not doxygen

Reply:
https://doxygen.nl/manual/docblocks.html#cppblock choice 3. Researching before claiming, please.

Reply:
I mean, we're not writing doxygen, there's no param/ret descriptions here, so what's the point of using a doxygen format?

Reply:
Those are doxygen, so if you want to use the same format please also complete the `@param` and `@return` part.